components/tidb_query/src/expr_util/collation/mod.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+mod utf8mb4;
```

Comment:
why `utf8mb4` ?

Reply:
What's your suggested name?

Reply:
`collate`?

Reply:
it will only contain utf8/utf8mb4 collations, in case of any future charsets.

Reply:
That is strange, why not contain 'bin'?

Reply:
because bin is placed in `mod.rs`.